.dockerignore

```diff
@@ -0,0 +1,31 @@
+/prometheus
+/thanos
+/vendor/
+/mixin/vendor/
+node_modules/
+
+# Ignore minikube setup working dirs.
+kube/bin
+kube/.kube
+kube/.minikube
```

Comment:
Do we use Kubernetes in our tests?

Reply:
I have copied those from `.gitignore`. I haven't checked them to be honest.